src/components/checkbox/checkbox.scss

```diff
@@ -6,16 +6,23 @@ $checkbox-border-width: 2px !default;
 $checkbox-text-margin: 10px !default;
 $checkbox-top-left: 12px !default;
 
+.md-inline-form {
```

Comment:
For each of the newly added `.md-inline-form` rules, can you add a brief description of the style changes that make the element sit in-line?